### PR TITLE
Update running-codeql-cli-in-your-ci-system.md

### PR DESCRIPTION
**Pull request by skhalife**
Updating to fix typo `upload` to `upload-results`

### Why:

Updating to fix typo `upload` to `upload-results`

### What's being changed:

A simple text change to fix typo in the CodeQL CLI upload-results example.

### Check off the following:
- [ ] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [ ] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [ ] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (eg. a "before and after video")

### CODE REVIEW

**Comment by janiceilene** [comment]
@skhalife Thanks so much for opening a PR! I'll get this triaged for review ⚡

**Review by felicitymay — APPROVED** [review]
Thank you so much for this fix 💖 

Very much appreciated.

**Review by benxoxo — COMMENTED** [review]
d11e30074871ad75dc91ecaf9144c2f3df177f11

File: content/code-security/secure-coding/running-codeql-cli-in-your-ci-system.md [hunk 1]
```diff
@@ -201,7 +201,7 @@ For more information, see [github upload-results](https://codeql.github.com/docs
 ##### Basic example
 
 ```
-$ echo $UPLOAD_TOKEN | codeql  github upload --repository=my-org/example-repo \
+$ echo $UPLOAD_TOKEN | codeql  github upload-results --repository=my-org/example-repo \
```
Comment (benxoxo) [inline content/code-security/secure-coding/running-codeql-cli-in-your-ci-system.md:204]: d11e30074871ad75dc91ecaf9144c2f3df177f11

**Review by benxoxo — COMMENTED** [review]
d11e30074871ad75dc91ecaf9144c2f3df177f11

**Pull request merged**